### PR TITLE
SAV-44: Add a blog post about the Durham Restaurants map

### PR DESCRIPTION
**Pull request by timstallmann**
This PR adds a blog post about the Durham Restaurants map. It also adds new .blog-image classes, with .wrap-left (for text to wrap around on the left) and .wrap-right (for text to wrap around on the right) options.

### CODE REVIEW

**Comment by timstallmann** [comment]
Not sure what our review process is for blog posts.. @AnneTee do you know?

**Comment by AnneTee** [comment]
@timstallmann We don't really have one! I think it's crucial for us to have at least one other person read and sign off on each post. Perhaps we can request whoever we think would be best (knows the most about the topic, has the most free time, knows that "flounder" and "founder" can be synonyms, etc.) to review the PR, then once we get their sign-off the post author can merge the code?

@kostajh @chrisarusso @dmurphy1 thoughts?

**Comment by kostajh** [comment]
@AnneTee your proposed review process sounds good to me!

**Comment by AnneTee** [comment]
@timstallmann Looks awesome, I just added markdownify to the post summary on /news and removed an apostrophe from your post :+1:

**Comment by timstallmann** [comment]
Glad to finally have this live! Thanks everyone for feedback/reviewing!

File: _posts/2015-04-27-durham-restaurant-time-machine.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<p>The Pan Pan Diner, at its original location on Hillandale Road (misspelled as <i>Hillondale</i> in the health inspections dataset)</p>
+</div>
+
+With this tool, you can click back through the years (from 1995 through 2015) to see which restaurants were open when, and where.
+No longer do you have to wonder:
+
+* Where was the original Pan Pan Diner located? (Hillandale Rd and I-85)
+* Where was Ninth Street Bakery before it was on W Chapel Hill Street, and when did it move (776 Ninth Street, 1997)
+* Is it true what people say that there were no restaurants downtown in the 1990s? (not exactly, but there were a lot fewer)
+
```
Comment (dmurphy1) [inline _posts/2015-04-27-durham-restaurant-time-machine.md:23]: @timstallmann - should this say "Is it true when people say"? I'm honestly not sure.

File: _posts/2015-04-27-durham-restaurant-time-machine.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<br><b>Click here to [take a look!](http://www.savaslabs.com/durham-restaurants-map)</b>
+And we'd love to hear what you think! You can reach us on twitter at <a href="https://twitter.com/savas_labs">@savas_labs</a>, or email <a href="mailto:info@savaslabs.com">info@savaslabs.com</a>. 
+
+### Sources
+The data comes from [Durham's new open data initiative](http://data.dconc.gov/), which provided a dataset of restaurant inspections going back all the way to 1990.
+We cleaned their data and mapped it using [Leaflet](http://www.leafletjs.com), an open-source Javascript mapping framework -- how we did that
```
Comment (dmurphy1) [inline _posts/2015-04-27-durham-restaurant-time-machine.md:29]: @timstallmann - maybe we should keep this in the present tense and go with "which provides" if the dataset is still available.

**Pull request merged**